launchlib/launcher.go

```diff
@@ -21,13 +21,26 @@ import (
 	"path"
 	"strings"
 	"syscall"
+	"regexp"
 )
 
 const (
-	TemplateDelimsOpen  = "{{"
+	TemplateDelimsOpen = "{{"
 	TemplateDelimsClose = "}}"
+	// Matches characters disallowed in paths we allow to be passed to exec()
+	ExecPathBlackListRegex = `[^\w.\/_\-]`
```

Comment:
I don't think we expand `~` to the home directory (this is a shell feature, right), so this won't have the intuitive effect anyway?